roles/addons/powerman/templates/powerman.conf.j2

```diff
@@ -14,20 +14,23 @@ include "/etc/powerman/ipmipower.dev"
   {% endfor %}
 {% else %}
   {% set range = groups['all'] %}
-{% endif %}
+{%- endif %}
 
-{% for equipment in j2_equipment_groups_list %}
-  {% set hosts = [] %}
-  {% set bmcs = [] %}
-  {% for host in groups[equipment] %}
-    {% if host in range and hostvars[host]['bmc']['name'] is defined %}
-      {{ hosts.append(host) }}
-      {{ bmcs.append(hostvars[host]['bmc']['name']) }}
-    {% endif %}
-  {% endfor %}
-  {% if hosts | length > 0 %}
+{%- for equipment in j2_equipment_groups_list %}
+  {%- set hosts = [] -%}
+  {%- set bmcs = [] -%}
+  {%- for host in groups[equipment] %}
+    {%- if host in range and hostvars[host]['bmc']['name'] is defined %}
+      {{- hosts.append(host) }}
+      {{- bmcs.append(hostvars[host]['bmc']['name']) }}
+    {%- endif %}
+  {%- endfor %}
+  {%- if hosts | length > 0 %}
+
+# {{ equipment }}
 device "{{ equipment }}" "ipmipower" "/usr/sbin/ipmipower --wait-until-on --wait-until-off -h {{ bmcs|join(',') }} -u {{ hostvars[hosts[0]]['ep_equipment_authentication']['user'] }} -p {{ hostvars[hosts[0]]['ep_equipment_authentication']['password'] }} |&"
 node "{{ hosts|join(',') }}" "{{ equipment }}" "{{ bmcs|join(',') }}"
-  {% endif %}
-{% endfor %}
+
+  {%- endif %}
+{%- endfor %}
```

Comment:
These `-` are not needed.

Reply:
I disagree:

- First dash for endif removed adds two lines between each section
- The dash for ednfor can be removed.